src/components/Dashboard/Dashboard.tsx

```diff
@@ -32,9 +38,39 @@ export const Dashboard = withRouter(props => {
   /**
    * Set token in token context
    */
-  tokenContext.setToken(getCookie("sessionToken"));
+  useEffect(() => {
+    /**
+     * Set token in token context
+     */
+    tokenContext.setToken(getCookie("sessionToken"));
+    console.log("setToken called!", tokenContext.token);
```

Comment:
I'd add a pivotal task TODO if this is a draft

Reply:
it was a console.log to check correct token use, i removed console.log call